terra-env/variables.tf

```diff
@@ -135,7 +136,23 @@ variable "wsm_db_keepers" {
   default     = false
   description = "Whether to use keepers to re-generate instance name. Disabled by default for backwards-compatibility"
 }
+variable "wsm_buffer_pool_names" {
```

Comment:
Do we ned to make `wsm_buffer_pool_names` and `wsm_external_folder_ids` as map?

Reply:
Thought we decided yesterday that that was too complicated. They actually address disjoint cases buffer_pool_names is for when WSM talks to RBS in it's own environment, while wsm_external_folder_ids is when it doesn't.